lib/oli_web/live/community_live/index.ex

```diff
@@ -7,33 +7,39 @@ defmodule OliWeb.CommunityLive.Index do
   alias OliWeb.Common.{Breadcrumb, Filter, Listing}
   alias OliWeb.CommunityLive.{New, TableModel}
   alias OliWeb.Router.Helpers, as: Routes
+  alias Surface.Components.Form
+  alias Surface.Components.Form.{Checkbox, Field, Label}
   alias Surface.Components.Link
 
-  data title, :string, default: "Communities"
-  data breadcrumbs, :any
+  data(title, :string, default: "Communities")
+  data(breadcrumbs, :any)
 
-  data filter, :string, default: ""
-  data total_count, :integer, default: 0
-  data offset, :integer, default: 0
-  data limit, :integer, default: 20
-  data sort, :string, default: "sort"
-  data page_change, :string, default: "page_change"
-  data show_bottom_paging, :boolean, default: false
-  data additional_table_class, :string, default: ""
```

Comment:
As @mcass19 mentioned elsewhere - I don't really like the extra parentheses here, would love to not have them as long as that's consistent with rest of liveviews